docs/install-upgrade/install.md

```diff
@@ -33,28 +33,45 @@ The main steps to install Fabric are:
     1. Boot them into ONIE Install Mode to have them automatically provisioned
 
 ## Build Control Node configuration and Installer
-Hedgehog has created a command line utility, called `hhfab`, that helps generate the wiring diagram and fabric configuration, validate the supplied configurations, and generate an installation image (.img) suitable for writing to a USB flash drive or mounting via IPMI virtual media. The first `hhfab` command to run is `hhfab init`. This will generate the main configuration file, `fab.yaml`. `fab.yaml` is responsible for almost every configuration of the fabric with the exception of the wiring. Each command and subcommand have usage messages, simply supply the `-h` flag to your command or sub command to see the available options. For example `hhfab vlab -h` and `hhfab vlab gen -h`.
+Hedgehog has created a command line utility, called `hhfab`, that helps generate the wiring diagram and fabric configuration,
+validate the supplied configurations, and generate an installation image (.img or .iso) suitable 
+for writing to a USB flash drive or mounting via IPMI virtual media. The first `hhfab` command to
+run is `hhfab init`. This will generate the main configuration file, `fab.yaml`. `fab.yaml` is 
+responsible for almost every configuration of the fabric with the exception of the wiring. Each 
+command and subcommand have usage messages, simply supply the `-h` flag to your command or sub 
+command to see the available options. For example `hhfab vlab -h` and `hhfab vlab gen -h`.
 
 ### HHFAB commands to make a bootable image
 
 1. `hhfab init --wiring wiring-lab.yaml`
 1. The `init` command generates a `fab.yaml` file, edit the `fab.yaml` file for your needs
     1. ensure the correct boot disk (e.g. `/dev/sda`) and control node NIC names are supplied
 1. `hhfab validate`
-1. `hhfab build`
+1. `hhfab build --mode iso`
+    1. There are two options for the format of the bootable media: ISO and USB. An ISO is best suited to use with IPMI based
+       virtual media. The USB file is designed for writing to a removable USB drive. ISO is the default.
 
-The installer for the fabric is generated in `$CWD/result/`. This installation image is named `control-1-install-usb.img` and is 7.5 GB in size. Once the image is created, you can write it to a USB drive, or mount it via virtual media.
+The installer for the fabric is generated in `$CWD/result/`. This installation image is named `control-1-install-usb.iso` and is 7.5 GB in size. Once the image is created, you can write it to a USB drive, or mount it via virtual media.
 
 ### Write USB Image to Disk
 
 !!! warning ""
     This will erase data on the USB disk.
 
+### Steps for Linux
```

Comment:
I meant to comment on that yesterday but it seems it got lost, I probably failed to add the comment to my review before validating it. So:

Do you know whether MkDocs support tabs, something like [in this example](https://docs.cilium.io/en/stable/gettingstarted/k8s-install-default/#install-the-cilium-cli) (but this is using Sphinx)? Would be nice to use here. Not blocking for this PR, though.

Reply:
https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#linked-content-tabs
something like this seems to be possible.